aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -928,6 +1006,35 @@ void convert_indices_from_csr_to_coo_cpu(
         }
       });
 }
+template <typename input_t, typename output_t>
+void compressed_to_batched_compressed_indices(
```

Comment:
@pearu This is the compressed -> batched compressed conversion kernel I spoke to you about. The key benefit is that rather than starting a kernel for each batch to convert `coo -> csr/csc` we can strategically reshape the target and generate coo indices for a matrix, generate `ccol/crow_indices` for that matrix and use this to convert to the `crow/ccol_indices` for the batched result.

So there would be 2 kernel launches rather than `1 + product(batch_dims)` kernels.